Package.swift

```diff
@@ -31,15 +31,15 @@ let package = Package(
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
         .package(url: "https://github.com/IBM-Swift/HeliumLogger.git", from: "1.7.0"),
-        .package(url: "https://github.com/IBM-Swift/Kitura-net.git", from: "2.1.0"),
+        .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "0.0.0"),
```

Comment:
I think this now needs to be `from: "1.0.0"` as Kitura-NIO has recently been tagged at that major